data/drak/drak ships.txt

```diff
@@ -32,7 +32,7 @@ ship "Archon"
 		"outfit space" 1300
 		"weapon capacity" 1125
 		"engine capacity" 200
-		"energy capacity" 20000
+		"energy capacity" 25000
```

Comment:
Spreading the energy capacity increase across the various outfits seems unnecessarily opaque. If you want to increase the Archon's energy capacity then just increase it here.

Reply:
Dispersed is more interesting because it says something about their outfits and technology rather than just "the ship has a big battery".

Also the very real problem that the so definition is stored, so just adding to that only affects new saves. Outfits are reloaded every time, so changing them means everyone's archons get the capacity increase without needing compatibility missions or anything. ,(There majority of the increase, anyway)

Reply:
Yeah for the same reason (I think) inscrutable is on one of the outfits (ships being saved in savefile really makes a mess sometimes)
This is kinda Korath style of each outfit has a battery

Reply:
Simply adding it to the Archon itself does feel like a much cleaner way of accomplishing the same task. However, the benefit of old saves still seeing the increase without a patch mission is a real tangible benefit, which in my opinion, does outweigh the extra "cleanliness" of adding the capacity directly to the Archon.

Reply:
Except that's not much of a concern given that the mission repeats now. The reason we had to have a patch mission before is because the old version of the mission wasn't designed to repeat. It had a `kill` objective, which meant that you'd have to land on the planet that the mission offered on in order for it to repeat, which is something that could theoretically never happen. Now, as soon as you kill any of the Archons and then land, they respawn, meaning any buff we gave them will appear.

Reply:
Ok, so we are back to the main reason that I think the implications of Archons having significant distributed power storage is much more conceptually interesting than then simply having a big built-in battery. I find that rather boring.

Reply:
What about it is interesting? Given how everything else is, it seems to me that these weapons are only their own outfits because they have to be. Trying to distribute energy capacity because that makes it more "interesting" just seems a bit silly when everything but the hyper and jump drive (for some reason) are completely built into the base attributes.

Reply:
The lack of anything else of interest is no reason to not add a detail that is intentionally being added to imply something of interest. If anything, it is more of a reason to do so.

Reply:
What does it imply that would be of interest?

What about putting energy capacity on various different weapons, which for all we know, are only actually discrete parts because it is not possible for the weapons to be build directly into the ship definition, is interesting?

Reply:
Of they were just discrete parts, they probably wouldn't have outfit masses and sizes either, or descriptions.

Reply:
Just to be clear, this is also content that will be relevant to future Ember Waste story.

Reply:
> I find it incredibly frustrating to have to argue such minutia so extensively. 

If you feel this detail is important enough to be worth arguing for. It is perfectly acceptable for others to find it important enough to warrant arguing against. 

If you truly believe this change is minute, inconsequential and doesn't matter either way, then simply commit the change and you can stop arguing about it. 

If you're instead saying, "this is a small change just do what I want", that is very close to stating "Just accept my decisions unquestionably."

Which I disagree with.

Reply:
>Just to be clear, this is also content that will be relevant to future Ember Waste story.

How, exactly?
The Archons having more energy by having it built-in achieves the same buff as if spreading it out, and is simpler overall.
If you're concerned about Lifted Lorax and Sleeping Dragon, you can just readd them in via the missions you're concerned in order to forcefully apply the buff to older saves.

Reply:
We need to put things in the outfits more, not less, if two solutions have the same gameplay aspect but one in simpler in that it doesnt require failing any missions and such, just use the simpler solution.

Reply:
Why do we need to put more things in outfits? Specifically, why do we gotta do that to outfits that are never even supposed to be seen by (as Archons can't be scanned), much less obtained by the vanilla player?
As Amazinite said, not like the buff requires any mission resets anyway, given Archons are immortal now they'd either just self-apply the buff upon respawn in case they're beat, or not be beat and at that point it doesn't matter. Mission Archons like Lifted Lorax would get the buff regardless.

Reply:
Because regardless of whether or not the player can scan them, people still read the data files and that still is part of the canonical lore of the game. Thus I want it to support story in the future.

Reply:
Yes.

Reply:
Can we get an in-game mention of that somewhere? Someone from a relevant faction mentioning this little fact?

Reply:
That will be coming, yes. Not immediately, sadly, but the whole point of this is to set things up for future content.

Reply:
> Yes.

Ok so I'm way more confused now. How are the stats of Archon outfits part of the lore?
How is it any relevant to the lore that the way we buff Archon energy cap is by specifically adding to its outfits? For all it's worth, the Archon's entire kit is part of its body, we just can't integrate weapons to a ship hull the same way we do its HP or speed values.

What in the future would hinge on Archon outfits having this energy cap?

Reply:
I'm sorry, but I don't feel like laying out several chapters worth of missions in a GitHub comment box to satisfy your curiosity.